apps/studio/prisma/migrations/20250329080947_add_site_id_fk_to_audit_log/migration.sql

```diff
@@ -0,0 +1,5 @@
+-- AlterTable
+ALTER TABLE "AuditLog" ADD COLUMN     "siteId" INTEGER;
+
+-- AddForeignKey
+ALTER TABLE "AuditLog" ADD CONSTRAINT "AuditLog_siteId_fkey" FOREIGN KEY ("siteId") REFERENCES "Site"("id") ON DELETE SET NULL ON UPDATE CASCADE;
```

Comment:
nit - instead of `ON DELETE SET NULL`, i think we should use `ON DELETE RESTRICT` to prevent Site record from being deleted. This will prevent accidental deletion of site by isoengineers, since we currently don't have any safeguards against accidental deletion of Site + we don't log site deletion

Reply:
sounds good